raiden/api/python.py

```diff
@@ -461,22 +481,27 @@ def set_total_channel_deposit(
             channel_id=channel_state.identifier,
         )
 
-        balance = token.balance_of(self.raiden.address)
+        token_network_balance = token.balance_of(registry_address)
```

Comment:
Nitpick/Suggestion: Balance is used for channels, since  token network is a collection of channels it may make more sense to call this `total_escrow` or `total_network_balance`